pkg/cortex/cortex_test.go

```diff
@@ -87,5 +87,5 @@ func TestCortex(t *testing.T) {
 	require.NotNil(t, serviceMap[Server])
 	require.NotNil(t, serviceMap[Ingester])
 	require.NotNil(t, serviceMap[Ring])
-	require.NotNil(t, serviceMap[Distributor])
+	require.NotNil(t, serviceMap[DistributorService])
```

Comment:
"All" target still includes `Distributor`, I think we can keep the previous version of the test.

Reply:
The new `distributor` target doesn't return a service so this was failing.